app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :pizzeria
+
+  validates :title, presence: true
+  validates :body, presence: true
```

Comment:
I would restrict the length on body. The `text` data type allows for huge strings so you'll want to restrict what the user can save.